### ISSUE TITLE
Initial Repository Clone

### ISSUE THREAD

**Issue by rolandf** [zeapo/Android-Password-Store#61]
I had to include the username in the entry field dedicated to the server such as this:

Username@bitbucket.org

to get a working url usable by jgit to get the initial repository clone done.

**Comment by rolandf** [comment]
I gave an other try, this time inputing the bitbucket info with protocol https at initial repository clone step. It didn't worked. The clone button didn't worked.
Then I exited the setting from the inital clone option. Next I clicked settings, my settings where saved and then only it started working when I retried to clone.
Clone went OK, then I made a new push from my laptop and then a pull from remote in Android-Password-Store and it worked.

Basically I played with the app until I got what I wanted it's hard to remember what I made exactly that made the app work for me.

Do you know an Android screen recorder like Camtasio or Screenflow ?
I'm willing to go through all this one more time and record it if it can help you.

**Comment by zeapo** [comment]
On Mon, Jan 5, 2015 at 11:55 PM, Frédéric Roland <notifications@github.com>
wrote:

> Do you know an Android screen recorder like Camtasio or Screenflow ?
> I'm willing to go through all this one more time and record it if it can
> help you.
>

​
https://play.google.com/store/apps/details?id=com.iwobanas.screenrecorder.noroot.free

That would be great :)​



-- 
Mohamed Zenadi

mail: mohamed@zenadi.com
webpage: http://zenadi.com

**Comment by Aster89** [comment]
Hello!
I think I have a similar problem that I'm not able to find a solution to.
When inserting "Repository information" what should I use to fill the fields? I've set the following
```
Protocol: https://
Username: MyUsername
Server URL: MyUsername@bitbucket.org
Repo path: MyUsername/MyRepo.git
```
which results in the correct "Resulting URL", `MyUsername@bitbucket.org/MyUsername/MyRepo.git`, _i.e._ the same URL I read when I run `pass git remote -v` in the shell, without the `http://` in front of it.

**Comment by Aster89** [comment]
It finally turned out that [deleting the password repository folder and re-creating](https://github.com/zeapo/Android-Password-Store/issues/357#issuecomment-349833720) it was enough (apparently I was wrong in thinking that I had done it already).
